samples_project/Assets/SampleViewer/Samples/Routing/RouteManager.cs

```diff
@@ -97,10 +97,12 @@ async void Update()
                     if (results.Contains("error"))
                     {
                         DisplayError(results);
+                        GameObject.Find("Text_Minutes").SetActive(false);
```

Comment:
This is very nit picky but to save you from writing this line twice you could set it outside the for loop and rewrite it to `GameObject.Find("Text_Minutes").SetActive(!results.Contains("error"));'

Also I personally find it risky to reference game objects by name in case later down the line the names are changed then the code will return null errors with the developer usually not knowing why, but it should not be TOO much of an issue...

Reply:
Good call! Yeah this error will show up when you put the APi key directly in the Project Settings instead.